NuGet/Realm.DataBinding/Realm.DataBinding.nuspec

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata minClientVersion="2.8">
+    <id>Realm.DataBinding</id>
+    <version>$version$</version>
+    <title>Realm.DataBinding</title>
+    <authors>Realm</authors>
+    <owners>Realm</owners>
+    <developmentDependency>false</developmentDependency>
+    <licenseUrl>https://opensource.org/licenses/Apache-2.0</licenseUrl>
+    <projectUrl>http://github.com/realm/realm-dotnet</projectUrl>
+    <iconUrl>http://realm.io/img/favicon-32x32.png</iconUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>Realm is a mobile database: a replacement for SQLite and ORMs</description>
```

Comment:
Yes, I didn't feel like coming up with something, so I just copy-pasted the old one.